composer.json

```diff
@@ -34,7 +34,8 @@
         "symfony/console":  "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.0"
+        "phpunit/phpunit":    "~5.0",
+        "mikey179/vfsStream": "^1.6"
```

Comment:
I get it. Still, I think it is put to good use in that case. But you're right, let's see what @maglnet has to say. 

And thanks for all the feedback!